app/signin/page.tsx

```diff
@@ -0,0 +1,134 @@
+"use client";
+import React, { useState } from "react";
+import styles from "../../styles/SigninPage.module.css";
+
+import { loginSuccess } from "../../utils/commonUtil";
+import { postReqeustApi } from "../../utils/requestApi";
+import SignInfoInputComponent from "../../components/SignInfoInputComponent";
+
+// email 정규식
+const regex = new RegExp("[a-z0-9]+@[a-z]+.[a-z]{2,3}");
```

Comment:
이렇게 표현해주면 주석을 따로 작성해주는 것 보다 더욱 명확할 것 같아요~
```suggestion
const EMAIL_REGEX = new RegExp("[a-z0-9]+@[a-z]+.[a-z]{2,3}");
```

Reply:
보통 이렇게 많이 표현하기도 해요!
```suggestion
// email 정규식
const regex = /[a-z0-9]+@[a-z]+.[a-z]{2,3}/;
```